tests/VolleyManagement.UnitTests/Services/GameReportService/GetPivotStandingsTests.cs

```diff
@@ -407,7 +467,49 @@ public void GetPivotStandings_PlannedGamesWithoutResults_StandingsContainPlanned
             TournamentStandings<PivotStandingsDto> actual,
             string message)
         {
-            AssertTournamentStandingsAreEqual(expected, actual, message, new PivotStandingsComparer());
+            PivotStandingsComparer comparer = new PivotStandingsComparer();
+            comparer.WithAllComparer();
```

Comment:
Why do we have to call this?

Reply:
if somebody wants to call all Comparers, we create a method that calls all lambda with all comparers

Reply:
Can we make this call implicitly?

Reply:
we can do this, but when we will call another comparer, we should turn off all another(in my example 3)

Reply:
Figure out best approach